authentication/esignet-integration-impl/src/main/java/io/mosip/authentication/esignet/integration/service/IdaAuditPluginImpl.java

```diff
@@ -54,6 +54,9 @@ public void logAudit(Action action, ActionStatus status, AuditDTO audit, Throwab
 
 	@Override
 	public void logAudit(String username, Action action, ActionStatus status, AuditDTO audit, Throwable t) {
+		if (t!=null && StringUtils.isEmpty(username)) {

```

Comment:
Why is this check and exception added? there can be audits which will not involve user.